pulse_actions/run_time_config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "exchange": "exchange/treeherder/v1/job-actions",
-    "topic": "buildbot.ash.#"
+    "exchange": "exchange/build/normalized",
+    "topic": "unittest.mozilla-inbound.#"
```

Comment:
Can we listen to two different exchanges and have different actions associated to each one?